src/Illuminate/Auth/AuthManager.php

```diff
@@ -122,7 +123,7 @@ public function createSessionDriver($name, $config)
     {
         $provider = $this->createUserProvider($config['provider'] ?? null);
 
-        $guard = new SessionGuard($name, $provider, $this->app['session.store']);
+        $guard = new SessionGuard($name, $provider, $this->app['session.store'], null, new Timebox);
```

Comment:
Maybe use the 'timebox' named parameter and remove the null parameter for better readability

Reply:
Thanks for the suggestion! Unforturnatly security PR's must go to the 8.x branch which goes down to php7.3, so can't use that in this case.